jupyterhub_singleuser_profiles/images.py

```diff
@@ -47,7 +47,10 @@ def get_form(self, last_image=None):
             self.get_images_legacy(result, last_image)
         else:
             for i in imagestream_list.items:
-                self.append_option(i, result, last_image)
+                if name_only:
```

Comment:
Do you need this condition? Can you just pass in the name_only and let the append_option check True/False (as it does anyway)